packages/@markuplint/rules/src/attr-duplication/index.ts

```diff
@@ -1,9 +1,9 @@
 import { createRule } from '@markuplint/ml-core';
 
 export default createRule({
-	async verify({ document, report, t }) {
+	verify({ document, report, t }) {
 		const message = t('{0} is {1:c}', t('the {0}', 'attribute name'), 'duplicated');
-		await document.walkOn('Element', node => {
+		void document.walkOn('Element', node => {
```

Comment:
`void` is used to workaround for [`@typescript-eslint/no-floating-promises`](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/no-floating-promises.md).

Reply:
Nice!

If so, could you add to the rule for ESLint?

Reply:
The default rule is https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/src/rules/no-floating-promises.ts#L55, but the CI is broken, I'll check ASAP.